site/en/docs/extensions/mv3/known-issues/index.md

```diff
@@ -57,11 +49,21 @@ This section lists major features that will be added to the Manifest V3 platform
 
 **Tracking issue:** [1247690][]
 
-### Service workers are not started in response to webRequest events
+## Fixed but not yet in Stable {: #resolved }
+
+### webRequest.onAuthRequired events {: #webrequest-onauthrequired }
+
+**Summary:** User-installed Manifest V3 extensions cannot currently intercept `webRequest.onAuthRequired` events, since the `webRequestBlocking` permission is restricted to policy-installed extensions. Chrome will provide extensions with a way to supply credentials for authentication requests in Manifest V3.
```

Comment:
```suggestion
**Summary:** User-installed Manifest V3 extensions cannot currently intercept `webRequest.onAuthRequired` events, since the `webRequestBlocking` permission is restricted to policy-installed extensions. Chrome provides extensions with a way to supply credentials for authentication requests in Manifest V3.
```

Reply:
Ack.

Reply:
Please commit this change. We're providing a way in the future. We're providing it now.

Reply:
I see. This got picked up in another correction. Never mind.